charger/easee.go

```diff
@@ -500,3 +507,40 @@ func (c *Easee) Identify() (string, error) {
 
 	return c.rfid, nil
 }
+
+// Set smart charging status to update the chargers led (smart=blue, fast=white)
+func (c *Easee) updateSmartCharging() {
+	if c.lp == nil {
+		return
+	}
+
+	mode := c.lp.GetMode()
+	isSmartCharging := mode == api.ModePV || mode == api.ModeMinPV
+
+	c.mux.L.Lock()
+	updateNeeded := isSmartCharging != c.smartCharging
+	c.mux.L.Unlock()
+
+	if updateNeeded {
+		c.log.DEBUG.Printf("update smart charging status: %v", isSmartCharging)
+
+		data := easee.ChargerSettings{
+			SmartCharging: &isSmartCharging,
+		}
+		uri := fmt.Sprintf("%s/chargers/%s/settings", easee.API, c.charger)
+		resp, err := c.Post(uri, request.JSONContent, request.MarshalJSON(data))
+		if err != nil {
+			c.log.WARN.Printf("failed to update smart charging status")
```

Comment:
Ich würde immer den echten Fehler loggen- sonst fragst du dich hinterher immer warum… und in dem Fall wirklich als Fehler? Http Status (auch 400, 404) willst du ignorieren- oder ist die Post Methode schon unsere?